ex3.py

```diff
@@ -0,0 +1,23 @@
+import decorator
+
+
+@decorator.decorator
+def twice(func, *args, **kwargs):
+    """
+    Decorator that executes the given function twice.
+    :param func: function to execute
+    :param args: argument for the function.
+    :param kwargs: dictionary for the function.
+    :return: decorator.
+    """
+    func(*args, **kwargs)
+    func(*args, **kwargs)
```

Comment:
A decorator should always return the function you decorate. Try running your code with a visible change in the console (printing for example) and see where it fails.